Lombiq.Tests.UI/Exceptions/CounterThresholdException.cs

```diff
@@ -0,0 +1,50 @@
+using Lombiq.Tests.UI.Services.Counters;
+using System;
+using System.Text;
+
+namespace Lombiq.Tests.UI.Exceptions;
+
+// We need constructors with required informations.
+#pragma warning disable CA1032 // Implement standard exception constructors
```

Comment:
The warning does not disallow custom constructors, but it enforces implementation of all default constructors incl. one to support serialization.

I don't think we should opt out of that.

Reply:
I concur, you must have [these constructors](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/quality-rules/ca1032#rule-description) for safe error handling, but you can have additional constructors if you want to include custom logic.